packages/blade/src/components/ActionList/ActionListItem.tsx

```diff
@@ -139,7 +135,11 @@ const _ActionListSection = ({
       >
         {children}
       </BaseBox>
-      {_hideDivider && isReactNative() ? null : <ActionListSectionDivider />}
+      {_hideDivider && isReactNative() ? null : (
+        <Box marginX="spacing.3" marginY="spacing.1">
```

Comment:
Why have we added these margins here?

Reply:
The [original ActionListSectionDivider](https://github.com/razorpay/blade/blob/84847e1511084b2bdd6f019b79e08e5398f13251/packages/blade/src/components/ActionList/ActionListItem.tsx#L85) component has margins, have tried to replicate that.